wcfsetup/install/files/lib/system/package/PackageUpdateDispatcher.class.php

```diff
@@ -630,45 +630,35 @@ protected function savePackageUpdates(array &$allNewPackages, $packageUpdateServ
                     }
                 }
 
-                // @deprecated 5.2
-                if (isset($versionData['compatibility'])) {
-                    foreach ($versionData['compatibility'] as $version) {
-                        $compatibilityInserts[] = [
-                            'packageUpdateVersionID' => $packageUpdateVersionID,
-                            'version' => $version,
-                        ];
+                // The API compatibility versions are deprecated, any package that exposes them must
```

Comment:
I would keep a deprecation comment here because at one point in the future, all compatibility code can be removed.